apps/backend/api/graphql/makeModels.js

```diff
@@ -60,6 +60,42 @@ export default function makeModels (userId, isAdmin, apiClient) {
       fetchMany: () => CommonRole.fetchAll()
     },
 
+    ContextWidget: {
+      model: ContextWidget,
+      attributes: [
+        'id',
+        'title',
+        'type', 
+        'order',
+        'visibility',
+        'view',
+        'icon',
+        'created_at',
+        'updated_at',
+        'highlightNumber',
+        'secondaryNumber'
+      ],
+      relations: [
+        { group: { alias: 'ownerGroup' } },
```

Comment:
why the alias? group seems good enough

Reply:
Just because there is also viewGroup and I wanted to make sure its clear that there are two possible distinct relations to the group table